frontend/src/components/panels/Document.vue

```diff
@@ -1,70 +1,71 @@
 <template>
 	<LayoutCol :class="'document'">
 		<LayoutRow :class="'options-bar'">
-			<div class="left side">
-				<DropdownInput :menuEntries="documentModeEntries" v-model:selectedIndex="documentModeSelectionIndex" :drawIcon="true" />
+			<FloatingMenuToggleGroup>
+				<div class="left side">
+					<DropdownInput :menuEntries="documentModeEntries" v-model:selectedIndex="documentModeSelectionIndex" :drawIcon="true" />
```

Comment:
I'm frustrated at GitHub's diff here. The only change is that I wrapped everything in FloatingMenuToggleGroup which means everything else got indented.

Reply:
![image](https://user-images.githubusercontent.com/21245806/129235834-3ed6a545-d954-4042-be95-4160aa1f3bdb.png)
looks fine in reviewable

Reply:
GitHub's diffs are shockingly bad. I'm amazed I can't find more complaints about it on the internet; I would have thought people would have revolted by now having to live with such useless diffs. It's seriously one of the absolute worst parts of the entire GitHub experience.

Although I'm not a fan of Reviewable as a whole, I added it to our project just so we can see competent diffs. I avoid it for small code reviews because I can make do with GitHub's diffs and not complicate the code review process with the comment-based reviews. I use it for medium-sized reviews. For big reviews, I pull the branch and use VSCode's Git Graph plugin where I Ctrl+click the PR branch and master to view the diff, then I find the associated file in GitHub's code review area and write my comments on the respective line.

That said, in this particular case, you can just set GitHub's code review feature to ignore whitespace:

![](https://files.keavon.com/-/ConcreteProfitablePrayingmantid/capture.png)